library/std/src/thread/mod.rs

```diff
@@ -887,6 +879,23 @@ pub fn sleep(dur: Duration) {
 ///
 /// * It can be implemented very efficiently on many platforms.
 ///
+/// # Memory Orderings
```

Comment:
"Memory orderings" as a countable noun phrase sounds like referring to atomic memory orderings or specific executions, but this section only uses atomic memory orderings as an alternative model for explanation, i.e., "which atomic memory orderings we would use if we were to implement them by atomic memory operations".

```suggestion
/// # Memory Ordering
```

```suggestion
/// # Effect on Memory Ordering
```

```suggestion
/// # Interaction with Memory Model
```

Reply:
I would "Synchronization" (or something involving that noun) as section title -- basically with weak memory concurrency, the side-effects of an operation include not only which loads and stores it performs, but also which kinds of synchronization it performs with other threads (mostly which release/acquire edges get established), and that's what this section is documenting.